source/usr_section/usr_manual/gui.rst

```diff
@@ -267,4 +325,25 @@ use the |select_location| :superscript:`Select Cursor Location` button together
 * Coordinates of the selected location are shown in the :guilabel:`x` and :guilabel:`y` fields. You may change the coordinate system of the displayed
   coordinates via the |mActionSetProjection| :superscript:`Select CRS` button (e.g. for switching to lat/long coordinates).
 
+Windows
```

Comment:
Windows or Views?
@janzandr we are inconsistent here, also in the software itself. How should we name those?

Reply:
As mentioned in #17 --> Views